tasks/nms/install-debian.yml

```diff
@@ -27,10 +27,28 @@
   register: nms_install_state
   when: nms_cleanup_status is not defined
 
-- name: (Debian/Ubuntu) Make sure NGINX Instance Manager is running
+- name: (Debian/Ubuntu) NMS Modules
```

Comment:
Do you need to make sure the module needs to be started and enabled after installation? Screenshot is from the NGINX docs for **acm** install.
<img width="487" alt="image" src="https://user-images.githubusercontent.com/62568830/225956370-aa531f14-03cc-4d45-8025-f82cb0b7825b.png">

Reply:
Not sure if this will be beneficial for you or not but in case it is you can allow specific module names. And have some dictionary that help translate to os repo name and systemd unit name..

In `var/main.yml`
```
nms_module_vars: [
  acm,
  sm,
]

nms_module_vals:
  acm:
    repo_name: nms-api-connectivity-manager
    systemd_name: nms-acm
  sm:
    repo_name: nms-sm
    systemd_name: nms-sm # name unconfirmed
```

Then in users can specify something like `acm` for `nms_modules[0].name`.

Your call.

Reply:
Good call, I have implemented this as you have suggested